tests/test_environment_LocalDescriptors.py

```diff
@@ -404,15 +404,15 @@ def test_query_point_ne_points(self):
         # again later anyway.
         lc = freud.locality.AABBQuery(box, points)
         nl = lc.query(
-            points, dict(exclude_ii=True, num_neighbors=num_neighbors)
+            query_points, dict(exclude_ii=False, num_neighbors=num_neighbors)
         ).toNeighborList()
 
         ld = freud.environment.LocalDescriptors(l_max, mode="global")
         ld.compute((box, points), query_points, neighbors=nl)
 
         # Loop over the sphs and compute them explicitly.
-        for idx, (i, j) in enumerate(nl):
-            bond = box.wrap(points[j] - query_points[i])
+        for idx, bond in enumerate(nl.vectors):
```

Comment:
What is the point of enumerating here if we are going to overwrite `bond` in the next line?

Reply:
This is a mistake on my part that occurred during some manual testing. Fixed.